.github/workflows/lagom-template.yml

```diff
@@ -76,6 +76,10 @@ jobs:
             elif ${{ inputs.toolchain == 'GNU' }} ; then
               echo "host_cc=gcc-13" >> "$GITHUB_OUTPUT"
               echo "host_cxx=g++-13" >> "$GITHUB_OUTPUT"
+              # lto1 uses more RAM than exists in the known universe when building ladybird 
+              if ${{ inputs.build_preset == 'Distribution_CI' }} ; then
+                CMAKE_OPTIONS="$CMAKE_OPTIONS -DENABLE_LTO_FOR_RELEASE=OFF"
+              fi
```

Comment:
Could we set this as the sensible default in CMake instead?

Reply:
Fair enough. I moved this to CMake instead.